source/assets/css/regions/_banner.scss

```diff
@@ -3,10 +3,15 @@
 
   a {
     display: inline-block;
+  }
+
+  &__brand
+  
+  a {
```

Comment:
Should this `a` be nested (meaning a curly brace is missing) or is it meant to be a part of the `&__brand` selector? If so, we should bring that up to the same line.